app.py

```diff
@@ -0,0 +1,12 @@
+from flask import Flask
+
+app = Flask(__name__)
+
+
+@app.route("/")
```

Comment:
Routes should move to some other file where all the routes will be added to the flask app.

Reply:
Will do it later